doc/doc/AUTHORS

```diff
@@ -0,0 +1,51 @@
+Vega Strike
+http://vegastrike.sourceforge.net
```

Comment:
We've moved to using an auto-generated authors listing: https://github.com/vegastrike/Vega-Strike-Engine-Source/blob/master/script/build-authors.sh